### PR TITLE
added modals

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
127 0 0 1_5500_Documentation_modal_modal html

### CODE REVIEW

**Review by aatishpandey — APPROVED** [review]
Nicely structured code.
Good naming convention.
Overall looks good to merge.

File: Documentation/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <!-- <link rel="stylesheet" href="../../index.css" /> -->
+    <link rel="stylesheet" href="../../Stylesheets/main.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/6026902e47.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Stark Slider</title>
```
Comment (Sameony) [inline Documentation/modal/modal.html]: change the title of the document according to the contents

File: Stylesheets/components/modal.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    font-size: 1.5rem;
+    margin-bottom: 0rem;
+}
+
+.modal-icon{
+    padding:0rem
+}
+
+.modal-wrapper{
+    border: 0.1rem solid var(--gray-color);
```
Comment (Sameony) [inline Stylesheets/components/modal.css:15]: for such minute sizes, its recommended to use pixels rather than rems

**Review by Sameony — APPROVED** [review]
Loved the design. Suggested some minor changes, do go through them.

File: Documentation/modal/modal.html (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply Documentation/modal/modal.html]: will do it

File: Stylesheets/components/modal.css (hunk 2 above)
Comment (Priyanshi-Tiwari) [reply Stylesheets/components/modal.css:15]: okay

**Pull request merged**